docs/cloud/rahti2/concepts.md

```diff
@@ -302,20 +329,19 @@ OpenShift includes all Kubernetes objects, plus some extensions:
 * **ImageStream** objects abstract images and
   enrich them to streams that emit signals when they see that a new image is
   uploaded into them by e.g. BuildConfig.
-* **DeploymentConfig** objects create new [**ReplicationControllers**](../tutorials/elemental_tutorial.md#replicationcontroller) based on the new images.
 * **Route** objects connects a **Service** with the internet using _HTTP_.
 
 ### DeploymentConfig
 
 !!! Warning
 
-    Starting OKD 4.14, DeploymentConfig API is being deprecated. Only security-related and critical issues will be fixed in the future.  
-    More information [here](https://access.redhat.com/articles/7041372)
+    Starting OKD 4.14, DeploymentConfig API is being deprecated. Only security-related and critical issues will be fixed in the future.
+    More information [here](https://access.redhat.com/articles/7041372) and the [replacement guide for DeploymentConfig](https://developers.redhat.com/learning/learn:openshift:replace-deprecated-deploymentconfigs-deployments/resource/resources:convert-deploymentconfig-deployment).
 
```

Comment:
Do we need to update apiVersion in 
deploymentconfig.yaml to "apps.openshift.io/v1" and in 
buildconfig.yaml to "build.openshift.io/v1"

Reply:
fixed, thanks